XamarinCommunityToolkit/Views/SnackBar/Options/ToastOptions.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views.Options
+{
+	/// <summary>
+	/// Toast options
+	/// </summary>
+	public class ToastOptions
+	{
+		public ToastOptions() => Result = new TaskCompletionSource<bool>(false);
+
+		/// <summary>
+		/// Message options: Message, color, font
+		/// </summary>
+		public MessageOptions MessageOptions { get; set; } = DefaultMessageOptions;
+
+		public static MessageOptions DefaultMessageOptions { get; set; } = new MessageOptions();
+
+		/// <summary>
+		/// Background color.
+		/// </summary>
+		public Color BackgroundColor { get; set; } = DefaultBackgroundColor;
+
+		public static Color DefaultBackgroundColor { get; set; } = Color.Default;
+
+		/// <summary>
+		/// Is Right to left
+		/// </summary>
+		public bool IsRtl { get; set; } = DefaultIsRtl;
+
+		public static bool DefaultIsRtl { get; set; } = false;
+
+		/// <summary>
+		/// The duration for the SnackBar.
+		/// </summary>
+		public int Duration { get; set; } = DefaultDuration;
```

Comment:
Can we add the unit here, in the name of the property, or/and in the `summary` part?